packages/browser-utils/browser-utils.ts

```diff
@@ -0,0 +1,30 @@
+// See https://stackoverflow.com/a/58064481
+export const isIOS =
+  /iPad|iPhone|iPod/.test(navigator.platform) ||
+  (navigator.platform === 'MacIntel' && navigator.maxTouchPoints > 1);
+
+let observed = false;
+
+/* istanbul ignore next */
+export const detectIosNavbar = () => {
```

Comment:
Is there a need to export this?

Reply:
Currently not needed, can remove. In `vaadin-overlay` we run this also when opening.

Reply:
Removed the export.